jfr-daemon/src/main/java/com/newrelic/jfr/daemon/DaemonConfig.java

```diff
@@ -18,6 +18,7 @@
   static final String DEFAULT_JMX_HOST = "localhost";
   static final int DEFAULT_JMX_PORT = 1099;
   static final boolean DEFAULT_USE_SHARED_FILESYSTEM = false;
+  static final boolean DEFAULT_AUDIT_LOGGING = false;
   static final Duration DEFAULT_HARVEST_INTERVAL = Duration.ofSeconds(10);
   static final String DEFAULT_MONITORED_APP_NAME = "My Application";
```

Comment:
```suggestion
  static final int DEFAULT_JMX_PORT = 1099;

  private static final String DEFAULT_JMX_HOST = "localhost";
  private static final boolean DEFAULT_USE_SHARED_FILESYSTEM = false;
  private static final boolean DEFAULT_AUDIT_LOGGING = false;
  private static final Duration DEFAULT_HARVEST_INTERVAL = Duration.ofSeconds(10);
  private static final String DEFAULT_MONITORED_APP_NAME = "My Application";
```
Almost all of these can be private.